crates/biome_js_analyze/src/lint/suspicious/no_comment_text.rs

```diff
@@ -16,23 +22,54 @@ declare_lint_rule! {
     /// ### Invalid
     ///
     /// ```jsx,expect_diagnostic
-    /// const a3 = <div>// comment</div>;
+    /// <div>// comment</div>;
     /// ```
     ///
     /// ```jsx,expect_diagnostic
-    /// const a4 = <div>/* comment */</div>;
+    /// <div>/* comment */</div>;
     /// ```
     ///
     /// ```jsx,expect_diagnostic
-    /// const a5 = <div>/** comment */</div>;
+    /// <div>/** comment */</div>;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <div>text /* comment */</div>;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <div>/* comment */ text</div>;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <div>
+    ///     text
+    ///     // comment
+    /// </div>;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <div>
+    ///     // comment
+    ///    text
+    /// </div>;
+    /// ```
+    ///
+    /// ```jsx,expect_diagnostic
+    /// <div>
+    ///     /* comment */
+    ///     text
+    /// </div>;
     /// ```
     ///
     /// ### Valid
     ///
     /// ```jsx
-    /// const a = <div>{/* comment */}</div>;
-    /// const a1 = <div>{/** comment */}</div>;
-    /// const a2 = <div className={"cls" /* comment */}></div>;
+    /// <div>{/* comment */}</div>;
```

Comment:
You need to be warraped inside a FRAGMENT?

https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#document-the-rule

```suggestion
    /// ```jsx
    /// <>
    ///    <div>{/* comment */}</div>;
    ///    <div>{/** comment */}</div>;
    ///    <div className={"cls" /* comment */}></div>;
    ///    <div>text {/* comment */}</div>;
    ///    <div>{/* comment */} text</div>;
    /// </>
    /// ```
```

Reply:
@chansuke 
I don't know why it would be better to enclose it in fragments, but I fixed it in the meantime.

If you don't mind, could you please tell me what you mean by this change?
I checked the link you provided, but don't really understand it...
[65ce3fe](https://github.com/biomejs/biome/pull/3446/commits/65ce3fed352f75724a56729f298931ef2f2b10bf)

Reply:
In the documentation, it is stated:

> When adding valid snippets in the ### Valid section, you can use one single snippet.

With this jsx example, I believe it should be treated as one single snippet using <></>.

Ex:https://github.com/dyc3/biome/blob/ec663b9fee9afa8bf9dec2e8a2d554628561128f/crates/biome_js_analyze/src/lint/nursery/use_consistent_curly_braces.rs#L39-L44

Reply:
@chansuke 
thank you!
I understood, FRAGMENT is necessary.